src/components/StaticShareImage.tsx

```diff
@@ -111,7 +111,7 @@ export default function StaticShareImage({
           <div tw="flex mb-8">
             {/* eslint-disable-next-line */}
             <img
-              src={user.avatarURL}
+              src={user.avatarURL.replace(/.webp|.gif/, ".png")}
```

Comment:
Is this required? I think it's in the right format already because of your updates in SharePopup#L43-L46

Reply:
yeah I texted about this in [discord chat](https://discord.com/channels/558328638911545423/1127595753360076982/1131623858206679222). Maybe I was getting cached result, we can remove this.

Reply:
Yes I saw this and I'm pretty sure it's because of the cache so it's safe to remove it.